R/local_G.R

```diff
@@ -87,9 +93,13 @@
 #'
 #' @inheritParams compute_transition_probabilities
 #'
-#' @references Bivand, R.S., Wong, D.W.S. Comparing implementations of global and
+#' @references
+#' - Bivand, R.S., Wong, D.W.S. Comparing implementations of global and
```

Comment:
Is this cited somewhere?

Reply:
In the text above you mean?

Reply:
Yes, I did a search on the raw file.

Reply:
Sure I guess we don't need that reference. It's removed now!

Reply:
I approved so feel free to merge when this is done.